tfexec/providers_schema_test.go

```diff
@@ -10,20 +10,44 @@ import (
 func TestProvidersSchemaCmd(t *testing.T) {
 	td := testTempDir(t)
 
-	tf, err := NewTerraform(td, tfVersion(t, testutil.Latest012))
+	tf, err := NewTerraform(td, tfVersion(t, testutil.Latest014))
```

Comment:
I remember there were some format changes at some point - is it perhaps safer to test both 0.12 and >=0.14 which introduced the format change?